### ISSUE TITLE
Be able to deploy more than one function

### ISSUE THREAD

**Issue by kturc** [alawik/rhapsode#2]
User is only able to deploy a single function. By allowing multiple functions at once, one is able to achieve increased usability.

**Comment by alawik** [comment]
Will need to redo the Deploy() function so that data pertaining to flags and CLI arguments are passed beforehand.

```go
func Deploy(fxPath string, imageTag string) {}
```

To clean up `main.go` we might want to make a separate file for the CLI handling, such as `cli.go`.

`./rhapsode deploy -f ./functions/node/. -t rtest/node_func:v3`

Where `-f` is the flag for the path to the function, and `-t` is the image tag.

**Comment by alawik** [comment]
I think this issue is dependent on #1

**Comment by alawik** [comment]
> Will need to redo the Deploy() function so that data pertaining to flags and CLI arguments are passed beforehand.
> 
> ```go
> func Deploy(fxPath string, imageTag string) {}
> ```
> 
> To clean up `main.go` we might want to make a separate file for the CLI handling, such as `cli.go`.
> 
> `./rhapsode deploy -f ./functions/node/. -t rtest/node_func:v3`
> 
> Where `-f` is the flag for the path to the function, and `-t` is the image tag.

Didn't have to do this yet, got it to work through a jerry-rig.

**Comment by alawik** [comment]
In commit ce820455b4a429a4dac440ed88d66b31cf31472e I set it up where `os.Args[]` are required for the `fxPath`, `imageTag`, and `port`. This results in a runtime error if those arguments are not inputted in the CLI when using the deploy command.

Will need a way better solution later since this won't work outside of the proof-of-concept, but will close this issue for now. Since I solved it this way, it's not dependent on #1, but the realistic solution might be.
